### PR TITLE
Update Android build tools to latest version

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

File: build/android/build.gradle [hunk 1]
```diff
@@ -10,8 +10,8 @@ buildscript {
 apply plugin: "com.android.application"
 
 android {
-	compileSdkVersion 23
-	buildToolsVersion "23.0.3"
+	compileSdkVersion 25
+	buildToolsVersion "25.0.0"
```
Comment (est31) [inline build/android/build.gradle]: The latest version is 25.0.1

File: build/android/sqlite [hunk 2]
```diff
@@ -0,0 +1 @@
+sqlite-amalgamation-3150200
```
Comment (est31) [inline build/android/sqlite]: I think this is some unwanted artifact

**Comment by rubenwardy** [comment]
Done, it works for me

**Comment by est31** [comment]
Not sure how to handle this feature freeze wise though.

**Comment by nerzhul** [comment]
Okay for me for the release as we didn't use specific rights

**Comment by paramat** [comment]
Tried to merge but didn't apply:

Applying: Update Android build tools to latest version
error: patch failed: build/android/build.gradle:10
error: build/android/build.gradle: patch does not apply
Patch failed at 0001 Update Android build tools to latest version

**Comment by est31** [comment]
Works great on github.

**Pull request merged**